src/util/asyncWrapper.ts

```diff
@@ -0,0 +1,4 @@
+export const asyncWrapper = fn => (req, res, next) => 
```

Comment:
Here `next` should be optional, since not every middleware has it. Also, `req`, `res` and `next` could have their type annotated.

Reply:
Well the whole point of the wrapper is that if in the middleware an async function throws an error, the `asyncWrapper` passes it to Express' error handler. If `next` is optional, then a thrown error still could be unhandled.

Reply:
You're right. Can you add the type annotations like this (and we can merge it after):
```ts
import { Request, Response, NextFunction } from 'express'

export const asyncWrapper = (fn: Function) => (req: Request, res: Response, next: NextFunction) =>
  Promise
    .resolve(fn(req, res, next))
    .catch(next)
```